missingNumbers.js

```diff
@@ -0,0 +1,20 @@
+function findMissingNumbers(inputArray) {
+    var missingNumbers = [];
+    if (!Array.isArray(inputArray)) {
+        throw new Error('Invalid Input');
+    }
+    if (inputArray.length <= 1) {
+        throw new Error('array length too small');
+    }
+    inputArray.sort((a, b) => a - b)
+    for (var i = 1; i < inputArray.length; i++) {
+        if (inputArray[i] - inputArray[i - 1] != 1) {
```

Comment:
Change this condition from `!=` to `>`. By doing that we don't run anything in the if statement if the difference is1 or less